### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/writefreely/actions/runs/4336838662#summary-11774328155

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/writefreely/actions/runs/4336838747#summary-11774328293

---

- [x] I have signed the [CLA](https://phabricator.write.as/L1)

### CODE REVIEW

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Build container image, publish as Github-package
+name: Build container image, publish as GitHub-package
```
Comment (jsoref) [inline .github/workflows/docker-publish.yml:1]: brand

File: account.go [hunk 2]
```diff
@@ -504,7 +504,7 @@ func login(app *App, w http.ResponseWriter, r *http.Request) error {
 			// User has no email set, so check if they haven't added a password, either,
 			// so we can return a more helpful error message.
 			if hasPass, _ := app.db.IsUserPassSet(u.ID); !hasPass {
-				log.Info("Tried logging in to %s, but no password or email.", signin.Alias)
+				log.Info("Tried logging into %s, but no password or email.", signin.Alias)
```
Comment (jsoref) [inline account.go:507]: this has been quite controversial. I'm happy to drop this or any other change

File: scripts/upgrade-server.sh [hunk 3]
```diff
@@ -2,7 +2,7 @@
 ###############################################################################
 ##			writefreely update script			     ##
 ##								 	     ##
-##	WARNING: running this script will overwrite any modifed assets or    ##
+##	WARNING: running this script will overwrite any modified assets or   ##
```
Comment (jsoref) [inline scripts/upgrade-server.sh:5]: this was the only case of whitespace damage that I noticed and adjusted for

**Review by jsoref — COMMENTED** [review]
Some corrections automatically suggested by Google Sheets. All fault mine

**Review by thebaer — APPROVED** [review]

**Pull request merged**